packages/backend/src/routers/headlessBrowser.ts

```diff
@@ -57,4 +95,83 @@ if (process.env.IS_PULL_REQUEST === 'true') {
+        try {
+            const browserWSEndpoint = `ws://${process.env.HEADLESS_BROWSER_HOST}:${process.env.HEADLESS_BROWSER_PORT}`;
+            console.debug(`Headless chrome endpoint: ${browserWSEndpoint}`);
+            browser = await puppeteer.connect({
+                browserWSEndpoint,
+            });
+
+            const page = await browser.newPage();
+            await page.setExtraHTTPHeaders({
+                cookie: req.headers.cookie || '',
+            });
+
+            await page.setViewport({
+                width: 1400,
+                height: 768, // hardcoded
+            });
+
+            const blockedUrls = [
+                'headwayapp.co',
+                'rudderlabs.com',
+                'analytics.lightdash.com',
+                'cohere.so',
+                'intercom.io',
+            ];
+            await page.setRequestInterception(true);
+            page.on('request', (request: any) => {
+                const requestUrl = request.url();
+                if (blockedUrls.includes(requestUrl)) {
+                    request.abort();
+                    return;
+                }
+
+                request.continue();
+            });
+            await page.goto(url, {
+                timeout: 100000,
+                waitUntil: 'networkidle0',
+            });
+
+            const selector = isDashboard
+                ? '.react-grid-layout'
```

Comment:
Should the dashboard screenshot include the filters at the top?

Reply:
hmm...I feel like no? 

it's kinda like "TV mode" I think? Like, if you were showing someone a dashboard on a big screen, you probably wouldn't show the filters 🤔 Or if you were sharing a dashboard with someone, I probably wouldn't want to see the filters. 

I'd say keep the filters out (it looks nicer without them) and then if people request it, we can add them in!

Reply:
Yeah, I vote no filters!